src/tools/user-agent-explainer/index.ts

```diff
@@ -0,0 +1,11 @@
+import { Browser } from '@vicons/tabler';
+import { defineTool } from '../tool';
+
+export const tool = defineTool({
+  name: 'User agent explainer',
+  path: '/user-agent-explainer',
+  description: 'Detect Browser, Engine, OS, CPU, and Device type/model from User-Agent data',
+  keywords: ['user', 'agent', 'explainer', 'browser', 'engine', 'os', 'cpu', 'device', 'user-agent', 'client'],
+  component: () => import('./user-agent-explainer.vue'),
+  icon: Browser,
+});
```

Comment:
Can you rename the tool and files to `User-agent parser`

```suggestion
export const tool = defineTool({
  name: 'User-agent parser',
  path: '/user-agent-parser',
  description: 'Detect and parse Browser, Engine, OS, CPU, and Device type/model from an user-agent string.',
  keywords: ['user', 'agent', 'explainer', 'browser', 'engine', 'os', 'cpu', 'device', 'user-agent', 'client'],
  component: () => import('./user-agent-parser.vue'),
  icon: Browser,
});
```

Reply:
Will fix that with next commit.